internal/datastore/crdb/schema/indexes.go

```diff
@@ -48,3 +52,75 @@ var crdbIndexes = []common.IndexDefinition{
 	IndexRelationshipBySubjectRelation,
 	IndexRelationshipWithIntegrity,
 }
+
+// IndexingHintForQueryShape returns an indexing hint for the given query shape, if any.
+func IndexingHintForQueryShape(schema common.SchemaInformation, qs queryshape.Shape) common.IndexingHint {
+	if schema.IntegrityEnabled {
+		// Don't force anything since we don't have the other indexes.
+		return nil
```

Comment:
why not returning `IndexPrimaryKey` here like in `IndexForFilter`?

Reply:
I don't want to force any index selection at all

Reply:
Then add a constant to represent that. it's not obvious what the behavior is returning `nil`